components/infobox/wikis/formula1/infobox_unit_car_custom.lua

```diff
@@ -79,7 +80,13 @@ function CustomUnit:setLpdbData(args)
 		type = 'car',
 		image = args.image,
 		date = args.released,
-		extradata = mw.ext.LiquipediaDB.lpdb_create_json{},
+		extradata = mw.ext.LiquipediaDB.lpdb_create_json({
+			manufacturer = args.manufacturer,
+			engineprovider = args.engine,
+			season = args.season,
+			team = args.team,
+			designer = args.designer,
+		}),
```

Comment:
As always, I'm gonna ask if there is a usecase for the data stored?
(YAGNI)

Reply:
Iirc it is for this https://discord.com/channels/93055209017729024/268719633366777856/1209472320268271646